.github/workflows/contracts.yaml

```diff
@@ -0,0 +1,104 @@
+                  cache: "yarn"
+                  registry-url: https://registry.npmjs.org
+
+            - name: Install Foundry
+              uses: foundry-rs/foundry-toolchain@v1
+
+            - name: Install Dependencies
+              run: yarn
+
+            - name: Build
+              run: yarn build
+              working-directory: packages/contracts
+
+            - name: Generate localhost deployment
+              run: ./dump_localhost.sh
+              working-directory: packages/contracts
+
+            - name: Docker meta
+              id: meta
+              uses: docker/metadata-action@v4
+              with:
+                  images: |
+                      docker.io/sunodo/deployments,enable=${{ github.event_name != 'pull_request' }}
+                      ghcr.io/sunodo/deployments
+                  tags: |
+                      type=match,pattern=(.+)@(.*),group=2
+                      type=ref,event=pr
+                  labels: |
+                      org.opencontainers.image.title=Sunodo's deployments
+                      org.opencontainers.image.description=Sunodo's deployments in an image
+
+            - name: Set up QEMU
+              uses: docker/setup-qemu-action@v2
+
+            - name: Set up Docker Buildx
+              uses: docker/setup-buildx-action@v2
+
+            - name: Login to GitHub Container Registry
+              uses: docker/login-action@v2
+              with:
+                  registry: ghcr.io
+                  username: ${{ github.actor }}
+                  password: ${{ secrets.GITHUB_TOKEN }}
+
+            - name: Login to DockerHub
+              uses: docker/login-action@v2
+              with:
+                  username: ${{ secrets.DOCKERHUB_USERNAME }}
+                  password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+            - name: Build and push
+              uses: docker/bake-action@v3
+              with:
+                  workdir: packages/contracts
+                  files: |
+                      ./docker-bake.hcl
+                      ./docker-bake.platforms.hcl
+                      ${{ steps.meta.outputs.bake-file }}
+                  set: |
+                      *.cache-from=type=gha
+                      *.cache-to=type=gha,mode=max
+                  push: true
+
+            - name: Publish package
+              if: startsWith(github.ref, 'refs/tags/@sunodo/contracts@')
+              run: npm publish --access public
+              env:
+                  NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+            - name: Upload to release
+              uses: softprops/action-gh-release@v1
+              if: startsWith(github.ref, 'refs/tags/@sunodo/contracts@')
+              with:
+                  body_path: packages/contracts/CHANGELOG.md
+                  files: |
+                      packages/contracts/anvil_state.json
+                      packages/contracts/deployments/**
+                      packages/contracts/export/abi/*
```

Comment:
Where can I see what's the expected filename here, and how the files are organized inside?

Reply:
there is one file per network, using the name of the network.
```
localhost.json
goerli.json
sepolia.json
...
```